root/static/scripts/release-editor/fields.js

```diff
@@ -445,6 +446,10 @@ class Medium {
                 self.release.earliestYear() < mediumFormatDate);
     });
     this.confirmedEarlyFormat = ko.observable(this.hasTooEarlyFormat());
+    this.hasUselessMediumTitle = ko.computed(function () {
+      return /^(CD|Disc)\s*\d+/i.test(self.name());
```

Comment:
Maybe match more common medium formats if that is likely to happen too?

```suggestion
      return /^(Cassette|CD|Disc|DVD|Vinyl)\s*\d+/i.test(self.name());
```

Reply:
I've seen it much more often with CD, but I guess it can't hurt to extend it a bit, DVD especially of this sounds likely. :)